### PR TITLE
feat: add helmRepoAdd function

### PR DESCRIPTION
**Pull request by wai-calvin**

### CODE REVIEW

**Review by christopherswarren — DISMISSED** [review]
👍

File: src/util/helm.js [hunk 1]
```diff
@@ -49,11 +51,23 @@ const updateHelmChartVersion = async (chartPath, version) => {
...
 };
 
+const extractChartUrl = async (chartPath) => {
+    //extract dependency chart URL
+    const chartYamlFile = path.join(chartPath, "Chart.yaml");
+    const chartYaml = await fs.readFile(chartYamlFile);
+
+    const doc = new YAML(chartYaml.toString());
+
+    return doc.dependencies.repository;
```
Comment (mrparkers) [inline src/util/helm.js]: Isn't `dependencies` an array?

Comment (wai-calvin) [reply src/util/helm.js]: yea, we realized this later on. but now we're extracting information from the chart in a different way

**Review by christopherswarren — APPROVED** [review]
Have seen this run successfully against Brad's fork of envelope game. Looks good.

**Pull request merged**